app/views/articles/feed.rss.builder

```diff
@@ -41,7 +41,7 @@ xml.rss(:version => "2.0",
       xml.item do
         xml.title article.title
         xml.tag!("dc:creator", user.instance_of?(User) ? user.name : article.user.name)
-        xml.pubDate article.published_at.to_s(:rfc822) if article.published_at
+        xml.pubDate article.published_at.to_fs(:rfc822) if article.published_at
```

Comment:
This change looks exactly like the suggested fix in the deprecation. Thanks!

The xml response being built is used for rss feeds (see https://dev.to/feed for examle), and I don't see any tests around this endpoint (I might not have looked closely enough, but it wasn't in the places I would have expected to find them). We probably want to add a test to confirm whether this is making any changes in behavior.

A minimal spec/requests/articles/articles_feed_spec.rb file covering the response content (not parsing the xml or validating that the rss is readable, only asserting the article is in the feed and the date is formatted the way we expect):

```ruby
require "rails_helper"

RSpec.describe "ArticlesFeed", type: :request do
  let!(:article) { create(:article) }

  it "returns an rss feed with published article details" do
    get "/feed"

    expect(response.body).to include(article.title)
    expect(response.body).to include("<pubDate>#{article.published_at.to_fs(:rfc822)}</pubDate>")
  end
end
```

I checked with expecting both of these values in the response body, it showed no difference for me and passed both times (the first does give a deprecation warning)

```ruby
"<pubDate>#{article.published_at.to_s(:rfc822)}</pubDate>"
"<pubDate>#{article.published_at.to_fs(:rfc822)}</pubDate>"
```